Source/Renderer/Texture.js

```diff
@@ -89,6 +89,21 @@ define([
                         internalFormat = WebGLConstants.R32F;
                         break;
                 }
+            } else if (pixelDatatype === PixelDatatype.HALF_FLOAT) {
```

Comment:
Do any of the changes in these file also need to be made in `CubeMap`/`CubeMapFace`?